recipe/recipe.yaml

```diff
@@ -43,4 +43,6 @@ about:
 
 extra:
   recipe-maintainers:
+    - bdice
     - gmarkall
+    - jakirkham
```

Comment:
I probably cannot commit significant time to this. So will depend on Graham to drive this forward. If you have questions or run into issues, am happy to chat though

Reply:
Many thanks - as Vyas volunteered, I've switched your user ID for his on this PR.

Reply:
Sounds good. Thanks Graham! 🙏

Am now watching the feedstock. Though probably best to ping me if it is important (lest it get buried by all the other conda-forge notifications 😅)